modules/identity-provider-secret.adoc

```diff
@@ -17,7 +17,7 @@ Identity providers use {product-title} `Secret` objects in the `openshift-config
 +
 [source,terminal]
 ----
-$ oc create secret generic <secret_name> --from-literal=clientSecret=<secret> -n openshift-config
+$ oc create secret generic <secret_name> --from-literal=tls.crt=<secret> --from-literal=tls.key=<secret> -n openshift-config
```

Comment:
You file you changed is modules/identity-provider-secret.adoc, this is one file not only referenced by BasicAuth, also referenced by others like OpenID, for those, `...clientSecret=...` is correct and the change will make those have incorrect oc commands, e.g. your current preview https://deploy-preview-43279--osdocs.netlify.app/openshift-enterprise/latest/authentication/identity_providers/configuring-oidc-identity-provider.html

Reply:
You should use separate file name other than modules/identity-provider-secret.adoc for BasicAuth

Reply:
In "enterprise-4.10" or "main" branches, the separate file name for BasicAuth is identity-provider-secret-tls.adoc : https://github.com/openshift/openshift-docs/blob/main/modules/identity-provider-secret-tls.adoc . You should add such file against 4.7 / 4.6 branches.

Reply:
Thanks

> > Only for versions 4.6 and 4.7
> 
> Given only for them, why the PR targets "openshift:main" branch? Branch "main" means latest version. I think you should submit one PR targeting branch enterprise-4.7 and enterprise-4.6 instead.

Sorry about that and thank you for catching! I will fix the branches base to taget enterprise-4.6 and 4.7, as well as ensure the new oc commands are only referenced in the BasicAuth and will update as soon as I can.